src/frontend/components/UI/Sidebar/components/HeroicVersion/index.tsx

```diff
@@ -54,55 +61,63 @@ export default React.memo(function HeroicVersion() {
   const version = heroicVersion
 
   return (
-    <>
-      {((showChangelogModal &&
-        !hideChangelogsOnStartup &&
-        heroicVersion !== lastChangelogShown) ||
-        showChangelogModalOnClick) && (
-        <ChangelogModal
-          dimissVersionCheck
-          onClose={() => {
-            setShowChangelogModal(false)
-            setShowChangelogModalOnClick(false)
-            setLastChangelogShown(heroicVersion)
-          }}
-        />
-      )}
-      <span
-        className="heroicVersion"
-        role="link"
-        title={t(
-          'info.heroic.click-to-see-changelog',
-          'Click to see changelog'
+    <div className="heroicVersionContainer" data-tour="sidebar-version">
+      <>
```

Comment:
we don't need this fragment if we are adding the wrapper div, it should make the diff less complicated cause it won't change the indentation of everything inside

Reply:
removed